src/bench/util/wallet_common.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_BENCH_UTIL_WALLET_COMMON_H
+#define BITCOIN_BENCH_UTIL_WALLET_COMMON_H
+
+#include <wallet/context.h>
+#include <wallet/wallet.h>
+
+using wallet::CWallet;
+using wallet::DatabaseOptions;
+using wallet::WalletContext;
+using wallet::WalletDatabase;
```

Comment:
tiny nit: could wrap the functions around a "namespace wallet {}" instead.

Reply:
Dropped `wallet_common.h`, but still ended up doing this in `wallet_loading` and `wallet_balance`